sektionen/organisation/sidebar_en.md

```diff
@@ -8,11 +8,11 @@ The Computer Science chapter By-laws, always in the latest audited version.
 
 The Rules regulate everything that the By-laws don't and are easier to change.
 
-#### [Operational Plan](/en/organisation/verksamhetsplan)
+#### [Operational Plan](/sektionen/dokument?lang=en)
 
 The current operational plan.
 
-#### [The Chapter Budget](/en/organisation/budget)
+#### [The Chapter Budget](/sektionen/dokument?lang=en)
```

Comment:
Idea:

We could link specifically to the budget using
`https://datasektionen.se/sektionen/dokument?lang=en#chapter-budget`

And if the specific heading title changes, It should still work like it does now. 

Same thing for the operational plan